In [13]:
import numpy as np

#### 1.Известно, что генеральная совокупность распределена нормально
    со средним квадратическим отклонением, равным 16.
    Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
    если выборочная средняя M = 80, а объем выборки n = 256.

$\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$

2Ф (t_lambda) = lambda = 0.95

Ф (t_lambda) = lambda = 0.475

t_lambda = 1.96 (По таблице Лапласа по адресу http://matecos.ru/formuly/formuly-i-tablitsy/tablitsa-laplasa.html)

In [2]:
delta = 1.96 * 16 / (256 ** 0.5)
delta

1.96

In [3]:
(80 - delta, 80 + delta)

(78.04, 81.96)

#### Ответ: Доверительный интервал = (78.04, 81.96)

Внимательно будьте со 2 и 4 заданиями.Мы на вебинаре рассматривали случаи с известной сигмой.Когда сигма генеральной совокупности нам неизвестна, делаем все то же самое,только берем t-критерий (критерий Стьюдента), а сигму считаем из выборки.В материалах есть таблица для критерия Стьюдента. График плотности распределения Стьюдента симметричный

####    2.В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,
    получены опытные данные:
    6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
    Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
    оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
    значение с доверительной вероятностью 0,95.


In [14]:
mes = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [18]:
std1 = mes.std(ddof=1)
std1

0.4508017549014448

In [21]:
mean_ = mes.mean()
mean_

6.590000000000001

n = 10, v = n-1 = 9

$\alpha = (1-0.95)/2 = 0.025$

t = 2.262 (По таблице критерия Стьюдента)

In [22]:
delta2 = 2.262 * 0.451 / (10 ** 0.5)
delta2

0.32260355023526943

In [23]:
(mean_ - delta2, mean_ + delta2)

(6.267396449764731, 6.9126035502352705)

#### Ответ: Истинное значение Х расположено в интервале (6.27, 6.91)

####    3.Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
  Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
    оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.
    

H0: M = 17

H1: M > 17



In [9]:
std = 4 **0.5
std

2.0

In [10]:
U_nabl = ((17.5 - 17) / 2) * (100**0.5)
U_nabl

2.5

Найдём критическую точку для односторонней критической области по таблице Лапласа:

${\alpha}$ = 0,05

Ф(х) = (1 - 2 * ${\alpha}$) / 2 = 0.45

In [7]:
u_kr = 1.645

In [11]:
u_kr < U_nabl

True

#### Ответ: отвергаем H0 и принимаем H1 (средний диаметр подшипников больше 17 мм)

Внимательно будьте со 2 и 4 заданиями.Мы на вебинаре рассматривали случаи с известной сигмой.Когда сигма генеральной совокупности нам неизвестна, делаем все то же самое,только берем t-критерий (критерий Стьюдента), а сигму считаем из выборки.В материалах есть таблица для критерия Стьюдента. График плотности распределения Стьюдента симметричный

####    4. Продавец утверждает, что средний вес пачки печенья составляет 200 г.
    Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
    202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
    Известно, что их веса распределены нормально.
    Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?



In [24]:
weight = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

H0: M = 200
H1: M != 200

In [30]:
stdw = weight.std(ddof=1)
stdw

4.453463071962462

In [28]:
stdm = weight.mean()
stdm

198.5

In [31]:
U_nablw = ((198.5 - 200) / 4.45) * (10**0.5)
U_nablw

-1.0659362899443976

n = 10, v = n-1 = 9

$\alpha = (1-0.99)/2 = 0.005$

t = 3.250 (По таблице критерия Стьюдента)

t > Unablw (знак не учитываем)

Принимаем гипотезу H0

#### Ответ: утверждение продавца верно.